ttnn/cpp/ttnn/tensor/tensor_impl.cpp

```diff
@@ -537,6 +431,8 @@ std::string to_string(const BufferType& buffer, const tt::tt_metal::LegacyShape&
         }
     } else if (layout == Layout::TILE) {
         switch (shape.rank()) {
+            case 0: print_datum(ss, buffer[0]); break;
+            case 1: ss << "Unsupported Rank (1) for printing tensor with TILE_LAYOUT!";
```

Comment:
Why isn't rank 1 supported for TileLayout? Can a user want to put a scalar with shape {1} into a tile?

Reply:
Should be supported. No much difference between 1 and 1,1 for tile